models/reviews_model.js

```diff
@@ -77,3 +77,26 @@ exports.insertComment = (review_id, newComment) => {
       return comment.rows[0];
     });
 };
+
+exports.updateReview = (review_id, updatedReview) => {
+  if (!updatedReview.hasOwnProperty("inc_votes")) {
+    return Promise.reject({
+      status: 400,
+      msg: "Bad Request",
+    });
+  }
+  const { inc_votes } = updatedReview;
+  if (isNaN(review_id) || isNaN(inc_votes)) {
+    return Promise.reject({
+      status: 400,
+      msg: "Bad Request",
+    });
+  }
+  return checkExists("reviews", "review_id", review_id)
```

Comment:
Your use of `checkExists` I don't think is strictly necessary here - if you were to simply try and update the review votes, you can check how many rows were affected in the results object